examples/react/app/src/main.tsx

```diff
@@ -6,9 +6,23 @@ import * as vg from '@uwdata/vgplot';
 import App from './App';
 import './ui/table-styles.css'; // Import the new stylesheet
 
-// Perform the one-time, global Mosaic setup here.
-// This configures the coordinator that our React app will later consume.
-vg.coordinator().databaseConnector(vg.socketConnector('ws://localhost:3000'));
+// --- Switchable Mosaic Backend Configuration ---
+
+// Read the environment variable. Default to 'server' if not set.
+const backend = import.meta.env.VITE_MOSAIC_BACKEND || 'server';
```

Comment:
I think this should default to wasm, since that is the only option that works without running additional commands